app/web/src/features/communities/events/EventForm.tsx

```diff
@@ -180,10 +193,7 @@ export default function EventForm({
           fullWidth
           helperText={errors.title?.message || ""}
           id="title"
-          inputRef={(element: HTMLInputElement | null) => {
-            element?.focus();
```

Comment:
I guess this was giving initial focus to the title, why was it removed?

Reply:
It was a workaround for the editor bug where it was stealing focus. Also I found it was focusing on the field on submit sometimes and I had to hit submit again for things to go through properly... Not quite sure why though